README.rst

```diff
@@ -48,16 +48,23 @@ We recommend using Linux as Operating System when running PostgreSQL, for perfor
 Install
 -------
 
-Assuming you have installed a postgresql server, in your shell:
+Assuming you have installed a postgresql server, there are two ways to set up the QWAT database:
+- The first way is to create a empty database using the ``init_qwat.sh`` script available in the source code.
+- The second way is to restore the sample dump using ``pg_restore``.
+
+First way. Using the source code
```

Comment:
How about "Create an empty database with init_qwat.sh" as the section title? And "Create a database from the sample dump using pg_restore" as the title of the next section? Just suggestions…

Reply:
Yes. Done.